client/my-sites/stats/controller.jsx

```diff
@@ -202,15 +202,13 @@ export function site( context, next ) {
 	const chartTab = validTabs.includes( queryOptions.tab ) ? queryOptions.tab : 'views';
 
 	context.primary = (
-		<LoadStatsPage>
```

Comment:
Set traffic page free from paywall

Reply:
We still need the [usage from the hook](https://github.com/Automattic/wp-calypso/blob/d6cc465d4efef61460fe3de22b6ccbcfcd7954f6/client/my-sites/stats/stats-redirect/load-stats-page.tsx#L86-L96) inside `LoadStatsPage`. We could move the query elsewhere to remove the `LoadStatsPage` wrapper.